content/en/docs/components/training/mpi.md

```diff
@@ -66,15 +66,47 @@ kubectl kustomize base | kubectl apply -f -
 You can create an MPI job by defining an `MPIJob` config file. See [TensorFlow benchmark example](https://github.com/kubeflow/mpi-operator/blob/master/examples/v2beta1/tensorflow-benchmarks.yaml) config file for launching a multi-node TensorFlow benchmark training job. You may change the config file based on your requirements.
 
 ```
-cat examples/v2beta1/tensorflow-benchmarks.yaml
+cat examples/v2beta1/tensorflow-benchmarks/tensorflow-benchmarks.yaml
 ```
 
 Deploy the `MPIJob` resource to start training:
 
 ```
-kubectl apply -f examples/v2beta1/tensorflow-benchmarks.yaml
+kubectl apply -f examples/v2beta1/tensorflow-benchmarks/tensorflow-benchmarks.yaml
 ```
 
+## Scheduling Policy
+The MPI Operator supports the [gang-scheduling](/docs/components/training/job-scheduling#running-jobs-with-gang-scheduling).
+If you want to modify the PodGroup parameters, you can configure in the following:  
+
+```diff
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: tensorflow-benchmarks
+spec:
+  slotsPerWorker: 1
+  runPolicy:
+    cleanPodPolicy: Running
++   schedulingPolicy:
++     minAvailable: 10
++     queue: test-queue
++     minResources:
++       requests:
++         cpu: 3000m
++     priorityClass: high
+  mpiReplicaSpecs:
+...
+```
+
+In addition, those fields are passed to the PodGroup for the volcano according to the following: 
```

Comment:
Also indicate that if you don't set the fields, mpi-operator populates them based on .....

Reply:
agree